docker-compose.yml

```diff
@@ -10,14 +10,27 @@ services:
     build:
       context: .
       dockerfile: ./docker/Dockerfile.base
-    image: local/pontoon_base
+      args:
+        pythonversion: "${PYTHON_VERSION}"
+    image: local/pontoon_base_${PYTHON_VERSION}
+
+  # Pontoon python3 base image
+  webapp.python3:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.base
+      args:
+        pythonversion: "${PYTHON_VERSION}"
+    image: local/pontoon-py3
```

Comment:
@jotes I think now it is ok

Reply:
And we can write the image like this `image: local/pontoon_base_py_${PYTHON_VERSION}`

Reply:
@aniruddha2000 do we need a separate service at all if it looks the same as `webapp`?
I mean, this could reduce the size of PR.

Reply:
Since the Python version is handled in the Makefile, and passed down when building the images, there is no need to have any differentiated services here. This `webapp.python3` target can be removed.